create_rental.rb

```diff
@@ -2,20 +2,26 @@
 require './student'
 require './teacher'
 require './rental'
+require './person'
+require 'json'
 
 def create_rental
   list_person
   print 'ID of person: (Choose "ID" from the above list)'
   person_id = gets.chomp.to_i
-  person = @person.find { |p| p.id == person_id }
+  person = @person.find { |p| p['id'] == person_id }
+  person_hashed = Person.new(person['age'], person['name'], id: person['id'])
   list_book
   print 'book type: (Choose book from the above list)'
   book_title = gets.chomp
-  book = @books.find { |b| b.title == book_title }
+  book = @books.find { |b| b['title'] == book_title }
+  book_hashed = Book.new(book['title'], book['author'])
   print 'Enter rental date: '
   rental_date = gets.chomp
-  rental = Rental.new(rental_date, person, book)
+  rental = Rental.new(rental_date, person_hashed, book_hashed)
   @rentals << rental
+  # method to rentals student to 'books.json'
+  save_rentals_to_json(@rentals)
```

Comment:
- [ ] Hey Team! 👋 Stellar job on implementing the method for saving rentals to JSON. 📚 However, there's a slight deviation from the project requirement. 🚧 According to the specifications, we're supposed to save to JSON only on app exit, not immediately upon rental creation. 🔄 Kindly review the current implementation and make the necessary adjustments to align with the project's directive. Let's ensure we save that data only when the app bids us farewell. 🌟 Thanks for your attention to detail! 🙌